fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -111,6 +113,8 @@
 
     private static final Logger LOGGER = getLogger(FedoraResourceImpl.class);
 
+    private static final PropertyConverter propertyConverter = new PropertyConverter();
```

Comment:
@ajs6f, would it make sense to make a ticket for Singleton-izing `PropertyConverter`? If so, please do.